mimic/test/test_loadbalancer.py

```diff
@@ -111,25 +111,26 @@ def setUp(self):
         self.root = self.helper.root
         self.uri = self.helper.uri
 
-    def _create_loadbalancer(self, name=None, api_helper=None):
+    def _create_loadbalancer(self, name=None, api_helper=None, nodes=None):
```

Comment:
We have too many methods in mimic without documentation for their parameters.  Since you're changing the signature on this one, can I make you add `:param:` declarations for the parameters to say what they are, what type they should be, and importantly why they're all optional and what the behavior is if they're omitted?  (Thanks.)